src/quickstart/provider/aws/install-aws.md

```diff
@@ -1,76 +1,45 @@
-KubeDNS is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kube-dns
-kubernetes-dashboard is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
-Grafana is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
-InfluxDB is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb
-```
-
-If you see a list of targets like the one above, `helmc` can communicate with the Kubernetes master.
+Ensure the `kubectl` client is installed and can connect to your Kubernetes cluster.
 
 ## Add the Deis Chart Repository
 
-The [Deis Chart Repository](https://github.com/deis/charts) contains everything you
-need to install Deis onto your Kubernetes cluster, with a single `helmc install` command.
+The Deis Chart Repository contains everything you need to install Workflow onto your Kubernetes
+cluster, with a single `helm install deis/workflow` command.
 
-Run the following command to add this repository to Helm Classic:
+Run the following command to add this repository to Helm:
 
 ```
-$ helmc repo add deis https://github.com/deis/charts
+$ helm repo add deis https://charts.deis.com/workflow
 ```
 
-!!! note
-   You need [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed in order to run the command above
-
 ## Install Deis Workflow
 
-Now that you have Helm Classic installed and have added the Deis Chart Repository, install Workflow by running:
+Now that Helm is installed and the repository has been added, install Workflow by running:
 
 ```
-$ helmc fetch deis/workflow-v2.8.0            # fetches the chart into a
-                                              # local workspace
-$ helmc generate -x manifests workflow-v2.8.0 # generates various secrets
-$ helmc install workflow-v2.8.0               # injects resources into
-                                              # your cluster
+$ helm install deis/workflow
 ```
 
-!!! Experimental
-	Workflow can also be installed now using the [Kubernetes Helm][helm]. All the details that are needed for a production deployments like off-cluster storage, external registry etc., can be configured by passing an optional [values file][valuesfile] which overrides default values.
-
-
-    	$ helm repo add deis https://charts.deis.com/workflow  # add the workflow charts repo
-
-    	$ helm install deis/workflow --version=v2.8.0 --namespace=deis -f <optional values file>  # injects resources into your cluster
-
-  See also our section on [Workflow chart provenance](../../../installing-workflow/workflow-helm-charts.md#chart-provenance)
-
-Helm Classic will install a variety of Kubernetes resources in the `deis` namespace.
-You'll need to wait for the pods that it launched to be ready. Monitor their status
-by running:
+Helm will install a variety of Kubernetes resources in the `deis` namespace.
```

Comment:
A note here that applies to all mentions of `helm install deis/workflow` -- I believe it is mandatory to add `--namespace deis` to said command for the chart to be installed into the `deis` namespace (or else, `default` will be used)